chapters/ch13.md

```diff
@@ -1,40 +1,26 @@
 # 13. You want to build a new email anti-spam system. Your team has several ideas:
-->
-
+-> Bạn mong muốn xây dựng một hệ thống phòng chống email rác mới. Đội của bạn có một vài ý tưởng:
 
 * Collect a huge training set of spam email. For example, set up a “honeypot”: deliberately send fake email addresses to known spammers, so that you can automatically harvest the spam messages they send to those addresses.
-->
-
+-> Thu thập một tập huấn luyện lớn về email rác. Ví dụ như thiết lập một Honeypot: cố ý gửi các địa chỉ email giả đến những spammer đã biết, và bạn có thể thu thập các tin nhắn rác mà họ gửi đến địa chỉ đó một cách tự động.
 
 * Develop features for understanding the text content of the email.
-->
-
-
-* Develop features for understanding the email envelope/header features to show what set
-->
-
-
-of internet servers the message went through.
-->
+-> Phát triển những tính năng để hiểu được nội dung bằng văn bản trong email.
 
+* Develop features for understanding the email envelope/header features to show what set of internet servers the message went through.
+-> Phát triển những tính năng để hiểu được các đặc tính của phông thư/nhãn thư từ email nhằm hiển thị tập hợp các máy chủ internet mà thư đã đi qua.
 
 * and more.
-->
-
+->và nhiều hơn thế.
 
 Even though I have worked extensively on anti-spam, I would still have a hard time picking one of these directions. It is even harder if you are not an expert in the application area.
-->
-
+-> Mặc dù tôi đã kinh qua rất nhiều trong việc phòng chống email rác, tôi vẫn có một khoảng thời gian khó khăn khi lựa chọn hướng tiếp cận này. Điều này sẽ còn khó hơn nếu bạn không phải là một chuyên gia trong việc ứng dụng lĩnh vực này.
```

Comment:
Em dịch không sai, nhưng anh thấy hơi dài và không tự nhiên. Anh đề xuất dịch ý thứ hai của câu đầu như sau:
> , tôi vẫn sẽ gặp khó khăn khi phải chọn một trong các hướng đi trên.

Câu thứ hai "expert in the application area" anh nghĩ dịch là "chuyên gia trong lĩnh vực này" là đủ hiểu.